src/components/CommentList.jsx

```diff
@@ -12,7 +16,7 @@ const CommentList = ({ review_id, newCommentBool }) => {
       setCommentList(comments);
       setIsLoading(false);
     });
-  }, [review_id, newCommentBool]);
+  }, [review_id, changeCommentBool]);
```

Comment:
You need to update the comment list locally, then do the request not refetch the comments again.